test/unit-test/test_derivation.ml

```diff
@@ -28,6 +28,61 @@ let%expect_test "derivation" =
       (Conj (Empty Empty))))
   |}];
 
+  (*
+   * case 42 of
+   *   X when true -> X
+   * end
+   *)
+  print Context.empty (Case (Constant (Number 42), [(PatVar "x", Constant (Atom "true")), Var "x"]));
```

Comment:
nits: Erlang's variable names must begin with capital letter: `PatVar "x"` → `PatVar "X"`

Reply:
Certainly.  I forgot it.  I'll fix it.

Reply:
It's fixed in the commit https://github.com/kmizu/fialyzer/commit/5e96528fa4cb51325d8afff2d745776fc221dd6c